tests/unit/blueprints/addon-test.js

```diff
@@ -70,7 +70,6 @@ describe('blueprint - addon', function() {
     "ember-addon"\n\
   ],\n\
   "scripts": {\n\
-    "lint:js": "eslint ./*.js addon addon-test-support app blueprints config lib server test-support tests",\n\
```

Comment:
this seems wrong to me. the line should still exist, but look a little different 🤔

Reply:
No, I think it is correct. This test is only testing the changes programmatically made by the addon blueprint. If you take a look at the fixtures above (addon, app, MU app, etc) you will see that we still have the `lint:js` script present...

Reply:
oh, it's the diff... nevermind 🙈